### ISSUE TITLE
bootloader_build_combined - Dependencies

### ISSUE THREAD

**Issue by MP-Dataprobe** [usedbytes/rp2040-serial-bootloader#5]
I am currently having issues with using bootloader_build_combined function. 
I had added the following:
1. A simple blinky program application for testing purpose.
2. Modified CMakeLists.txt and added 
   add_executable(test"app.c")
   target_link_libraries(test pico_stdlib)
   bootloader_build_combined(test)

The build process results in an error upon trying to build the executable

Error : "No rule to make target '../test.bin', needed by 'test_app.bin'.  Stop."

Any help please?

**Comment by usedbytes** [comment]
Could you please share your full CMakeLists.txt?

**Comment by MP-Dataprobe** [comment]
Hello, here is the full CMakeLists.txt file.

[CMakeLists-BOOTLOADER.txt](https://github.com/usedbytes/rp2040-serial-bootloader/files/10531096/CMakeLists-BOOTLOADER.txt)

**Comment by usedbytes** [comment]
Right, it looks like I haven't made the `bootloader_build_combined` target build the app binary.

Could you try adding this line before line 104:
```
picowota_build_standalone(${NAME})
```

I've implemented a somewhat neater version of this build process in my Pico-W WiFi OTA bootloader: https://github.com/usedbytes/picowota/ , I should port that across or combine them really.
